.pre-commit-config.yaml

```diff
@@ -1,11 +1,11 @@
 repos:
 -   repo: https://github.com/pycqa/flake8
-    rev: 6.0.0  # New version tags can be found here: https://github.com/pycqa/flake8/tags
+    rev: 7.1.1  # New version tags can be found here: https://github.com/pycqa/flake8/tags
```

Comment:
Want there a corresponding dev requirement that you wanted to keep in sync?

Reply:
Good catch, I forgot that!